packages/@o3r/extractors/src/core/wrapper.ts

```diff
@@ -12,6 +17,43 @@ export const createBuilderWithMetricsIfInstalled: BuilderWrapper = (builderFn) =
   try {
     const { createBuilderWithMetrics } = await import('@o3r/telemetry');
     wrapper = createBuilderWithMetrics;
```

Comment:
The usage of the wrapper should not be driven by boolean `o3rMetrics`?
Because if the boolean is `false` (and maybe `undefined` as well?) it is misleading to report metrics if the package `@or3/telemetry` in found by node in any `node_module`